cloud-controller-manager/do/loadbalancers_test.go

```diff
@@ -863,6 +863,129 @@ func Test_buildForwardingRules(t *testing.T) {
 			},
 			nil,
 		},
+		{
+			"default forwarding rules",
+			&v1.Service{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test",
+					UID:  "abc123",
+					Annotations: map[string]string{
+						annDOStickySessionsType:       "cookies",
+						annDOStickySessionsCookieName: "DO-CCM",
+						annDOStickySessionsCookieTTL:  "300",
+					},
+				},
+				Spec: v1.ServiceSpec{
+					Ports: []v1.ServicePort{
+						{
+							Name:     "test",
+							Protocol: "TCP",
+							Port:     int32(80),
+							NodePort: int32(30000),
+						},
+					},
+				},
+			},
+			[]godo.ForwardingRule{
+				{
+					EntryProtocol:  "http",
+					EntryPort:      80,
+					TargetProtocol: "http",
+					TargetPort:     30000,
+					CertificateID:  "",
+					TlsPassthrough: false,
+				},
+			},
+			nil,
+		},
+		{
+			"http forwarding rules",
+			&v1.Service{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test",
+					UID:  "abc123",
+					Annotations: map[string]string{
+						annDOProtocol:                 "http",
+						annDOStickySessionsType:       "cookies",
+						annDOStickySessionsCookieName: "DO-CCM",
+						annDOStickySessionsCookieTTL:  "300",
+					},
+				},
+				Spec: v1.ServiceSpec{
+					Ports: []v1.ServicePort{
+						{
+							Name:     "test",
+							Protocol: "TCP",
+							Port:     int32(80),
+							NodePort: int32(30000),
+						},
+					},
+				},
+			},
+			[]godo.ForwardingRule{
+				{
+					EntryProtocol:  "http",
+					EntryPort:      80,
+					TargetProtocol: "http",
+					TargetPort:     30000,
+					CertificateID:  "",
+					TlsPassthrough: false,
+				},
+			},
+			nil,
+		},
+		{
+			"https forwarding rules with sticky sessions",
```

Comment:
Is the base of this test similar enough to ["tls forwarding rules"](https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/749a63e79dc08d4f1ed7f66d7fa69a7c748d1b04/cloud-controller-manager/do/loadbalancers_test.go#L675)that we may want to rename it to "tls forwarding rules with sticky sessions"? WDYT?